Competition/src/main/java/competition/commandgroups/drivecommandgroups/DriveToNearestLoadingStationCommandGroup.java

```diff
@@ -0,0 +1,34 @@
+package competition.commandgroups.drivecommandgroups;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.pose.PoseSubsystem;
+import competition.subsystems.pose.SetPoseToFieldLandmarkCommand;
+import competition.subsystems.pose.PoseSubsystem.FieldLandmark;
+import competition.subsystems.pose.PoseSubsystem.Side;
+import competition.subsystems.vision.commands.RotateToVisionTargetCommand;
+import xbot.common.command.BaseCommandGroup;
+import xbot.common.subsystems.drive.ConfigurablePurePursuitCommand;
+
+public class DriveToNearestLoadingStationCommandGroup extends BaseCommandGroup {
+
+    @Inject
+    public DriveToNearestLoadingStationCommandGroup(RotateToVisionTargetCommand rotate,
+            SetPoseToFieldLandmarkCommand setPose, PoseSubsystem pose,
+            ConfigurablePurePursuitCommand goToLoadingStation) {
+
+        if (setPose.getSide() == Side.Right) {
```

Comment:
This isn't going to work - the left/right properties are set at the moment of creation. We'll need to talk about how to do this.